### PR TITLE
Create rss feed widget

### PR DESCRIPTION
**Pull request by cbryant24**
Story: https://chapman.my.workfront.com/task/view?ID=5e7bc0c9027160dee25d5e0b5cb36014

### CODE REVIEW

File: app/assets/stylesheets/widgets/rss-feed-display.scss [hunk 1]
```diff
@@ -0,0 +1,123 @@
+.rss-feed-display-widget {
+  margin: 2rem 0;
+  .text--link {
+    border-bottom: 2px solid currentColor;
+  }
+
+  &__title {
+
```
Comment (nicholasnadel) [inline app/assets/stylesheets/widgets/rss-feed-display.scss:8]: should we excommunicate this guy

Comment (cbryant24) [reply app/assets/stylesheets/widgets/rss-feed-display.scss]: @nicholasnadel good catch,  just removed it

File: .cascade-code/Chapman.edu/_cascade/formats/modular/uninav/uninav.vtl [hunk 2]
```diff
@@ -294,7 +294,7 @@ On a page level, configure via:
 
       #if ( $isUmbrellaCategory )
         <span tabindex="0" class="toggle-menu-label">${umbrellaLabel} Menu</span>
-      #else
+      #elseif ( $currentPathRoot != "index" && $currentPathRoot != "")
```
Comment (corejh) [inline .cascade-code/Chapman.edu/_cascade/formats/modular/uninav/uninav.vtl:297]: can you explain this?

Comment (cbryant24) [reply .cascade-code/Chapman.edu/_cascade/formats/modular/uninav/uninav.vtl:297]: checking to make sure it's not the homepage which is just `index`

Comment (cbryant24) [reply .cascade-code/Chapman.edu/_cascade/formats/modular/uninav/uninav.vtl:297]: It's already live on production

**Pull request merged**